src/sentry/api/endpoints/project_details.py

```diff
@@ -129,6 +130,10 @@ def validate_allowedDomains(self, attrs, source):
 
     def validate_slug(self, attrs, source):
         slug = attrs[source]
+        if slug in RESERVED_PROJECT_SLUGS:
+            raise serializers.ValidationError(
+                'The slug "%s" is reserved and not allowed.' %
```

Comment:
Good point.

![image](https://user-images.githubusercontent.com/375744/39377374-c16b34d4-4a09-11e8-8c6f-4cebfdd6a11c.png)

@dcramer @ckj should I still call this slug, or "name"?

Reply:
Or "id" I guess.

Reply:
I'm just going to keep this as "slug" for now because that's what the other error messages that I copy/pasted from. If we want to change it to something else, we should just do it to all of them at the same time.